packages/stack/blockchain/src/index.js

```diff
@@ -30,17 +32,17 @@ export default class Blockchain {
     }
 
     this.blockchainNodes = {};
-    this.events.setCommandHandler("blockchain:node:register", (clientName, clientFunctions) => {
+    this.events.setCommandHandler("blockchain:node:register", (clientName, clientFunctions, cb = () => {}) => {
       const {isStartedFn, launchFn, stopFn, provider} = clientFunctions;
 
       if (!isStartedFn) {
-        throw new Error(`Blockchain client '${clientName}' must be registered with an 'isStarted' function, client not registered.`);
+        return cb(`Blockchain client '${clientName}' must be registered with an 'isStarted' function, client not registered.`);
```

Comment:
Should callback be passed `new Error('Blockchain client ...')` instead of a string?